cmd/clusterctl/cmd/backup.go

```diff
@@ -45,6 +45,7 @@ var backupCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) error {
 		return runBackup()
 	},
+	Deprecated: "use 'clusterctl move' instead.",
```

Comment:
nit
```suggestion
	Deprecated: "use 'clusterctl move --to-folder' instead.",
```
same in the docs